geonode/br/management/commands/settings.ini

```diff
@@ -3,7 +3,7 @@ pgdump = pg_dump
 pgrestore = pg_restore
 
 [geoserver]
-datadir = geoserver/data
+datadir = /geoserver_data/data/
```

Comment:
This depends on the local environment so the settings.ini should be overridable, or we should be able to pass a specific settingslini to the b/r commands. Is it possible right now @bieganowski ?

Reply:
yes, sure - in the `-c` / '--config` argument one can specify the `settings.ini` file's location

Reply:
ok, from what @afabiani the settings.ini with datadir = /geoserver_data/data/ should only be the default for geonode with docker. The case without GeoNode should be reverted to "geoserver/data", which in any case will probably need to be overridden locally.

Reply:
so, the proposal is to rename settings.ini to settings.ini.sample and create a settings_docker.ini.sample. 
The first one will have geoserver/data.
The latter will have the datadir as proposed in this patch.

Reply:
Shall we do this before merging?

Reply:
@bieganowski has created a new seprate issue for this, which will override (remove) this settings.ini, so we can merge this and then merge the next PR right after